index.html

```diff
@@ -420,6 +420,7 @@
 			.done(function(data, statusText, jqXHR)
 			{
 				var firmwareData = { "firmware": data.firmware };
+				var validUrl = true;
```

Comment:
This seems redundant - just do a check on line 484 for the `githubUrl` key.

Reply:
can't check on line 484 because i change the null value to (not available) on line 440